themes/gohugo-theme-rene/layouts/portfolio/list.html

```diff
@@ -5,7 +5,7 @@
     <aside class="ph3 flex-ns mr4 ml4 flex-wrap justify-around mt5">
       {{ range .Paginator.Pages }}
 
-        <div class="relative tracked w-100 w-30-l pa3 tc mb3 bg-white">
+        <div class="relative tracked w-100 w-40-l pa3 tc mb3 bg-white">
           {{ .Render "summary" }}
```

Comment:
Jeste sem muzes pridat:

```
<script type="text/javascript">window.location = "https://expeditions.renehlavova.com/"</script>
```

Ale koukam, ze `meta refresh` je podporovanej vsude, tak je to zbytecny, sorry :D https://caniuse.com/?search=meta%20refresh